jax/interpreters/ad.py

```diff
@@ -397,18 +397,19 @@ def add_tangents(x, y):
 
 
 def defvjp_all(prim, custom_vjp):
```

Comment:
For my understanding, how is this related to the defvjp_all in api.py?

Reply:
This one applies to primitives (i.e. its first argument must be a core.Primitive instance), while the one in api.py applies to `custom_transforms` functions. (Those two paragraphs that we deleted from the `custom_transforms` docstring explains how a Primitive is different from a custom_transforms function!)

Reply:
Oh right. I think I figured that out but forgot to delete this review comment, my bad!